indico/modules/events/registration/templates/display/_regform_info.html

```diff
@@ -36,7 +36,7 @@
                     <div class="label">
                         {% trans %}Registration fee{% endtrans %}
                     </div>
-                    <div>{{ regform.base_price }} {{ currency }}</div>
+                    <div>{{ regform.base_price }} {{ regform.currency }}</div>
```

Comment:
I'd add a `render_base_price` method to RegistrationForm like i did with `Registration`. In any case, `format_currency` to get proper after-decimal digits!